docs/user/codes/vasp.md

```diff
@@ -258,6 +258,33 @@ VASP_CMD: <<VASP_CMD>>
 LOBSTER_CMD: <<LOBSTER_CMD>>
 ```
 
+The corresponding flow could, for example, be started with the following code:
+
+```Python
+from jobflow import SETTINGS
+from jobflow import run_locally
+from pymatgen.core.structure import Structure
+
+from atomate2.vasp.flows.lobster import VaspLobsterMaker
+from atomate2.vasp.powerups import update_user_incar_settings
+
+store = SETTINGS.JOB_STORE
+structure = Structure(
+    lattice=[[0, 2.13, 2.13], [2.13, 0, 2.13], [2.13, 2.13, 0]],
+    species=["Mg", "O"],
+    coords=[[0, 0, 0], [0.5, 0.5, 0.5]],
+)
+
+lobster = VaspLobsterMaker().make(structure)
+
+# update the incar
+lobster = update_user_incar_settings(lobster, {"NPAR": 4})
+# run the job
+run_locally(lobster, create_folders=True, store=store)
+```
+
+
+It is, however,  computationally very beneficial to define two different types of job scripts for the VASP and Lobster runs.
```

Comment:
Not clear to me what is meant. Is it because you have double the max wall time with 2 jobs?

Reply:
Two types of parallelisations. MPI vs. OpenMP

Reply:
Actually something we could also fix in custodian. Currently you cannot change number of openmp threads as part of the VASP or Lobster command.

Reply:
@janosh tried to add the explanation. Cleaner now?

Reply:
Perfect!